test/src/.tool-versions

```diff
@@ -0,0 +1 @@
+golang 1.22.2
```

Comment:
I'd suggest moving it to the global `.tool-versions` in the root.
It's easier for users if there's a single source of truth.